lib/LaTeXML/Util/Pathname.pm

```diff
@@ -545,6 +556,13 @@ The type C<*> matches any extension.
 Like C<pathname_find>,
 but returns I<all> matching (absolute) paths that exist.
 
+=item C<< $path = pathname_kpsewhich(@names); >>
+
+Attempt to find a candidate name via the external C<kpsewhich>
+capability of the system's TeX toolchain. If C<kpsewhich> is
+not available, or the file is not found, returns a
+Perl undefined value.
```

Comment:
bonus documentation item: realized we hadn't included an entry for pathname_kpsewhich